### ISSUE TITLE
Recurring problem

### ISSUE THREAD

**Issue by MichaelSu-821** [d-ailin/GDN#8]
Hi,First of all, thank you very much for sharing the code, I am trying your code, the environment is configured according to your requirements(PyG=1.5.0,pytorch=1.5.1), using msl data to try to run normally, but when I use wadi data set for training, after each epoch Loss The values of ACU_loss and ACU_loss are both Nan(At least the first 15 epochs are all this result), which seems to be unusual (wadi has been modified to ensure that it is consistent with the msl format，for example, 0 means normal, 1 means attack. contains 127 sequences, and the code has not been changed). I would like to ask if you need to perform the data set Some preprocessing?
Best Regards

**Comment by MichaelSu-821** [comment]
if loss is Nan, the model cannot be saved, so this error will be reported
![train error](https://user-images.githubusercontent.com/86837984/135951509-195e817c-7885-4c87-ae5e-c4b8aed70285.jpg)

**Comment by d-ailin** [comment]
Hi, we use min-max normalization. Could you try this to see whether it could solve the issue? (I guess it might be a reason). For the other preprocessing, you could refer to our paper.
Thanks.
